docs/testplan.md

```diff
@@ -134,6 +134,7 @@ spec:
 
 - [ ] Install a 1-node cluster of previous LTS version.
 - [ ] Upgrade the cluster to the current version.
+  - [ ] Verify that after customizing the log forwarder resource, changes survive the upgrade.
```

Comment:
In this case, I would also add a step "Set up a log forwarder resource" as a sub-step of the first "install" step.